src/js/core.js

```diff
@@ -356,21 +356,20 @@ function MediumEditor(elements, options) {
         }
     }
 
-    function initPasteHandler() {
-        var pasteOptions = Util.extend(
-            {},
-            this.options.paste,
-            // Backwards compatability
-            {
-                forcePlainText: this.options.forcePlainText, // deprecated
-                cleanPastedHTML: this.options.cleanPastedHTML, // deprecated
-                disableReturn: this.options.disableReturn,
-                targetBlank: this.options.targetBlank,
-                contentWindow: this.options.contentWindow,
-                ownerDocument: this.options.ownerDocument
-        });
-
-        this.pasteHandler = new PasteHandler(this, pasteOptions);
+    function initPasteHandler(options) {
+        // Backwards compatability
+        var defaultsBC = {
+            forcePlainText: this.options.forcePlainText, // deprecated
```

Comment:
Agreed, let me see if there's an easy path to do that, or let me know if you have ideas.